### PR TITLE
Py3

### PR DESCRIPTION
**Pull request by jburel**
* Update script
* Bump omero version
cc @dominikl

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-plugins-push#148](https://idr1-slot3.openmicroscopy.org:48443/jenkins/job/OMERO-plugins-push/148/). See the [console output](https://idr1-slot3.openmicroscopy.org:48443/jenkins/job/OMERO-plugins-push/148/consoleText) for more details.
Possible conflicts:
  - PR #61 dominikl 'Mask rois'
    - DESCRIPTION

--conflicts

**Comment by dominikl** [comment]
Looks good, travis passed. Just not 100% sure if the correct OMERO version was actually used, due to these lines in travis output:
```
2020-01-13 16:23:14,026 335        [      main] INFO          ome.formats.importer.ImportConfig - OMERO Version: 5.5.5
2020-01-13 16:23:14,050 359        [      main] INFO          ome.formats.importer.ImportConfig - Bioformats version: 6.3.1 revision: e355a853c5768721711cb9b29c3a27561539c4e4 date: 13 December 2019
```

The OMERO version table in the Readme could be updated as well as part of this PR, eg. an entry for OMERO 5.6 / romero.gateway 0.4.8 . 

Apart from that I think it's good to merge.

**Comment by jburel** [comment]
that's the version of omero-blitz see https://github.com/ome/omero-blitz/pull/78

**Comment by dominikl** [comment]
👍 Of course that comes from blitz, thanks.

**Comment by dominikl** [comment]
Didn't know that I can add commits on this PR directly :-) Just added the 5.6 entry to the table in the Readme file.

**Comment by dominikl** [comment]
I think this is good to merge. Tested locally too:
```
> library(romero.gateway)
Loading required package: rJava
Loading required package: jpeg
Loading required package: httr
Have to download OMERO Java libraries https://downloads.openmicroscopy.org/latest/omero5.6/OMERO.java.zip
Proceed? [y/n]
y
Downloading...
Extracting...
Finished. OMERO Java libraries saved in /home/dominik/R/x86_64-pc-linux-gnu-library/3.6/romero.gateway/java

*** Welcome to rOMERO 0.4.8 ***
```

File: README.md [hunk 1]
```diff
@@ -26,8 +26,9 @@ Download the latest romero.gateway version which is compatible with your OMERO s
 
 OMERO version | romero.gateway version
 --- | --- 
+5.6 | 0.4.8
```
Comment (joshmoore) [inline README.md]: Does it also work with 5.5?

Comment (dominikl) [reply README.md]: I'm not sure, that's why I just put 5.6 there. But I'll test it, could be less restrictive then.

Comment (dominikl) [reply README.md]: It is compatible to 5.5. Updated Readme accordingly again.

**Comment by jburel** [comment]
@dominikl can you go ahead with releasing it? date will need to be modified

**Pull request merged**